substratevm/src/com.oracle.svm.agent/src/com/oracle/svm/agent/BreakpointInterceptor.java

```diff
@@ -942,6 +942,37 @@ private static boolean objectStreamClassConstructor(JNIEnvironment jni, Breakpoi
         return true;
     }
 
+    private static boolean customTargetConstructorSerialization(JNIEnvironment jni, @SuppressWarnings("unused") Breakpoint bp, InterceptedState state) {
```

Comment:
We need a comment here explaining what we catch and what we output.

Reply:
Provided a short description.